### PR TITLE
fix video layout in jazzbar and conversation space

### PR DESCRIPTION
**Pull request by denisdimitrov**
- fix video layout
- add modal when device is not found or it couldn't start the camera
- fix the wrong counting of the conversation space
- fix double press for conversation space bubbles

### CODE REVIEW

File: src/components/organisms/Room/Room.tsx [hunk 1]
```diff
@@ -52,6 +59,55 @@ const Room: React.FC<RoomProps> = ({
...
 
+  const connectToVideoRoom = () => {
+    if (!token) return;
+    setVideoError("");
+
+    Video.connect(token, {
+      name: roomName,
+    })
+      .then((room) => {
+        console.log("surrece", room);
```
Comment (IvanBuljovcic) [inline src/components/organisms/Room/Room.tsx:70]: Intentional console log?

File: src/components/organisms/Room/Room.scss [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+
 .participant-container {
   background: $black;
   display: flex;
   justify-content: center;
 }
 
 .participant-container {
-  width: 16%;
+  width: 33%;
```
Comment (thecadams) [inline src/components/organisms/Room/Room.scss:18]: Can't believe it was that simple 🤩

File: src/components/organisms/UserProfileModal/UserProfileModal.tsx [hunk 3]
```diff
@@ -265,7 +265,7 @@ const SuspectedLocation: React.FC<{ user: WithId<User> }> = ({ user }) => {
     () => ({
       venue: venues?.find(
         (v) =>
-          (user.lastSeenIn ?? user.lastSeenIn[venue?.name ?? ""]) ||
+          (user.lastSeenIn && user.lastSeenIn[venue?.name ?? ""]) ||
```
Comment (thecadams) [inline src/components/organisms/UserProfileModal/UserProfileModal.tsx:268]: great find.

File: src/components/templates/ConversationSpace/ConversationSpace.tsx [hunk 4]
```diff
@@ -35,7 +35,7 @@ const ConversationSpace: React.FunctionComponent = () => {
   const venueUsers = users
     ? users.filter(
         (user) =>
-          user.lastSeenIn ??
+          user.lastSeenIn &&
```
Comment (thecadams) [inline src/components/templates/ConversationSpace/ConversationSpace.tsx:38]: again - great find

**Review by thecadams — APPROVED** [review]
lgtm! Great stuff, some really important fixes in there too.

**Pull request merged**